streams/src/commonTest/kotlin/com/mirego/trikot/streams/reactive/processors/RejectProcessorTests.kt

```diff
@@ -0,0 +1,34 @@
+package com.mirego.trikot.streams.reactive.processors
+
+import com.mirego.trikot.streams.cancellable.CancellableManager
+import com.mirego.trikot.streams.reactive.Publishers
+import com.mirego.trikot.streams.reactive.reject
+import com.mirego.trikot.streams.reactive.subscribe
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class RejectProcessorTests {
+    @Test
+    fun rejectTrue() {
+        val publisher = Publishers.behaviorSubject("a")
+        var value: String? = null
+
+        publisher.reject { true }.subscribe(CancellableManager()) {
+            value = it
+        }
+
+        assertEquals(null, value)
+    }
+
+    @Test
+    fun rejectFalse() {
```

Comment:
We would also need a test that actually reject the value in the block, not just true or false.

Reply:
Not sure that this is what you mean, but having the test actually use the value to decide if it rejects or not instead of hardcoding `true` of `false` would be a better test.

Reply:
@gbourassa Yes, that's what I meant 😄 👍🏻